js/libs/core.js

```diff
@@ -83,7 +83,7 @@ Fliplet.FormBuilder = (function() {
        *
        * @return {void} - this method does not return anything but updates the value directly on the target field.
        */
-      component.methods.setValueFromDefaultSettings = function(data) {
+      component.methods.setValueFromDefaultSettings = async function(data) {
```

Comment:
@tonytlwu This is for getting the default value before getting the value from LFD.
While I was testing, I noticed that when the field default value is from user data and I'm trying to edit LFD entry, the default value is set later than the entry value.